product_variant_configurator/views/inherited_product_product_views.xml

```diff
@@ -36,28 +36,28 @@
                         name="product_attribute_ids"
                         nolabel="1"
                         colspan="2"
-                        invisible="id or not product_tmpl_id or not product_attribute_ids"
+                        invisible="id or not product_tmpl_id or product_attribute_ids == []"
                         context="{'default_product_template_id': product_tmpl_id, 'default_owner_model': 'product.product', 'show_price_extra': True, 'show_attribute': False}"
                     >
-                        <tree create="0" delete="0" editable="1">
-                            <field name="owner_model" column_invisible="1" />
-                            <field name="owner_id" column_invisible="1" />
+                        <list create="0" delete="0" editable="bottom">
+                            <field name="owner_model" invisible="1" />
+                            <field name="owner_id" invisible="1" />
```

Comment:
Why are you replacing these `column_invisible` by `invisible`? Both attributes exist but don't do the same :
https://www.odoo.com/documentation/18.0/developer/reference/user_interface/view_architectures/generic_attribute_column_invisible.html